cli/version/version.go

```diff
@@ -39,5 +43,29 @@ func NewCommand() *cobra.Command {
 }
 
 func run(cmd *cobra.Command, args []string) {
-	feedback.Print(globals.VersionInfo)
+	if strings.Contains(globals.VersionInfo.VersionString, "git-snapshot") || strings.Contains(globals.VersionInfo.VersionString, "nightly") {
+		// We're using a development version, no need to check if there's a
+		// new release available
+		feedback.Print(globals.VersionInfo)
+		return
+	}
```

Comment:
I don't know if this is sensible here... I'd leave the `version` command to always check for the latest version (even in dev builds)

```suggestion
```

Reply:
I can't, it would fail because `globals.VersionInfo.VersionString` can't be parsed as semver.